Source/Engine/PackageBuilder/SyntaxParser.cs

```diff
@@ -851,6 +905,9 @@ private Syntax ParsePatternReference(string patternName)
                     else
                         throw SyntaxError(TextResource.AttributesAreNotAllowedForStandardPattern, patternName);
                 }
+                else if (pattern.Body is TokenSyntax token)
+                    result = new TokenSyntax(token.TokenKind, token.Text, token.IsCaseSensitive, token.TextIsPrefix,
```

Comment:
To be able to add positions to TokenSyntax objects, it's necessary to create copy of ones defined in Syntax.StandardPattern class.